src/internal/kopia/base_finder.go

```diff
@@ -251,6 +252,8 @@ func (b *baseFinder) findBasesInSet(
 				logger.Ctx(ictx).Info("found incomplete backup")
```

Comment:
I think this if-block also needs updated since we're possibly adding to the set of assist backups here. Maybe removing this block and checking below for a non-empty incomplete reason (or requiring an empty incomplete reason) would lead to somewhat cleaner code?